Wrappers/WebPageElement.cs

```diff
@@ -0,0 +1,140 @@
+﻿using Functionality_Tests_Suit.Constants;
+using Functionality_Tests_Suit.FactoryPattern;
+using OpenQA.Selenium;
+using SeleniumExtras.WaitHelpers;
+using System.Collections.ObjectModel;
+using System.Drawing;
+using Utilities;
+
+namespace Wrappers
+{
+    public class WebPageElement : IWebElement
+    {
+        protected IWebDriver Driver = BrowserFactory.GetDriver(BrowserType.Chrome);
+        protected IWebElement Element;
+
+        public WebPageElement(IWebElement element)
+        {
+            Element = element;
+        }
```

Comment:
Let's make the constructor protected for sure that we use it only in WebPageElement class and inheritors